decidim-core/lib/decidim/core/engine.rb

```diff
@@ -224,6 +225,10 @@ class Engine < ::Rails::Engine
         ENV["SHAKAPACKER_CONFIG"] = Decidim::Webpacker.configuration.configuration_file
       end
 
+      initializer "decidim_core.active_storage_variant_processor" do |app|
+        app.config.active_storage.variant_processor = :mini_magick
```

Comment:
See the new defaults. We should migrate to libvips. We can do that in another PR.